src/ProjectSystemTools/BinaryLogEditor/ViewModel/BuildViewModel.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.Tools.BinaryLogEditor.ViewModel
 {
     internal sealed class BuildViewModel : NodeViewModel
     {
-        private readonly LogModel.Build _build;
+        private readonly ProjectSystem.LogModel.Build _build;
```

Comment:
Related to https://github.com/dotnet/project-system/pull/6563#discussion_r482493732
I would consider renaming this data type

Reply:
Should be renamed now :)